rebar.config.script

```diff
@@ -1,11 +1,21 @@
-CONFIG1 = case os:getenv("TRAVIS") of
-              "true" ->
-                  JobId = os:getenv("TRAVIS_JOB_ID"),
-                  [{coveralls_service_job_id, JobId},
-                   {coveralls_coverdata, "_build/test/cover/*.coverdata"},
-                   {coveralls_service_name , "travis-ci"} | CONFIG];
-              _ ->
-                  CONFIG
-          end,
+%% -*-: erlang -*-
 
-CONFIG1.
+case {os:getenv("GITHUB_ACTIONS"), os:getenv("GITHUB_TOKEN")} of
+  {"true", Token} when is_list(Token) ->
+    Env = fun(K) -> os:getenv(K) end,
```

Comment:
nit:  this only shortens the the call from `os:getenv` to `Env`, really necessary ?